lxd/device/disk.go

```diff
@@ -1012,3 +1047,65 @@ func (d *disk) getParentBlocks(path string) ([]string, error) {
 
 	return devices, nil
 }
+
+// generateVMConfigDrive generates an ISO containing the cloud init config for a VM.
+// Returns the path to the ISO.
+func (d *disk) generateVMConfigDrive() (string, error) {
+	configDrivePath := filepath.Join(d.instance.Path(), "configdrv")
```

Comment:
Probably ought to do that stuff within `devices` just in case someone is weird enough to setup more than one of those :)

Reply:
@stgraber makes sense, will.